### PR TITLE
(chore): update `genereate-manifests.ps1` to catch up recently changes

### LINKED ISSUE #191: `generate-manifests.ps1` is outdated
Currently, the [`generate-manifests.ps1`](https://github.com/matthewjberger/scoop-nerd-fonts/blob/ea89cb134a1d6c2e4df62c4970b1509177dc33cf/bin/generate-manifests.ps1) script is outdated.

It doesn't reflect the changes in #89 and #161.

### PR DESCRIPTION
**Pull request by jonz94**
Closes #191

### CODE REVIEW

File: bin/generate-manifests.ps1 [hunk 1]
```diff
@@ -1,51 +1,99 @@
...
+        "installer"   = @{
+            "script" = @(
+                '$currentBuildNumber = [int] (Get-ItemProperty "HKLM:\SOFTWARE\Microsoft\Windows NT\CurrentVersion").CurrentBuildNumber',
+                '$windows1809BuildNumber = 17763',
+                '$isPerUserFontInstallationSupported = $currentBuildNumber -ge $windows1809BuildNumber',
+                '$isFontInstallationForAllUsers = $global -or !$isPerUserFontInstallationSupported',
+                'if ($isFontInstallationForAllUsers -and !(is_admin)) {',
+                '    error "Administrator rights are required to install $app."',
+                '    exit 1',
+                '}'
```
Comment (matthewjberger) [inline bin/generate-manifests.ps1:51]: might have missed a comma here 🙏🏼

Comment (jonz94) [reply bin/generate-manifests.ps1:51]: Oops... you're right!
But the script works as expected when i tested it, the output manifest doesn't have missing comma on that line.
So stange 🤔

Comment (jonz94) [reply bin/generate-manifests.ps1]: Okay, I learn a new thing about powershell today.

It turns out that **if we declare an array on multiple lines, the comma is optional**.

Reference: https://docs.microsoft.com/en-us/powershell/scripting/learn/deep-dives/everything-about-arrays?view=powershell-7.2#create-an-array

screenshots from powershell docs

So the missing comma will not effect the behavior of the script 🤣

Comment (matthewjberger) [reply bin/generate-manifests.ps1]: TIL! Thank you! :)

**Pull request merged**